modules/lvms-provisioning-storage-using-logical-volume-manager-operator.adoc

```diff
@@ -4,67 +4,68 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="lvms-provisioning-storage-using-lvms_{context}"]
-= Provisioning storage using {lvms}
+= Provisioning storage
 
-You can provision persistent volume claims (PVCs) using the storage class that is created during the Operator installation. You can provision block and file PVCs, however, the storage is allocated only when a pod that uses the PVC is created.
+After you have created the LVM volume groups using the `LVMCluster` custom resource (CR), you can provision the storage by creating persistent volume claims (PVCs).
+
+To create a PVC, you must create a `PersistentVolumeClaim` CR.
 
 [NOTE]
 ====
-{lvms} provisions PVCs in units of 1 GiB. The requested storage is rounded up to the nearest GiB.
+{lvms} provisions storage in units of 1 GiB. The requested storage is rounded up to the nearest GiB.
```

Comment:
From LVMS 4.15 onwards, PVCs can also provision PVs with size lesser than 1GiB with lvms operator.

Reply:
@radeore , @jakobmoellerdev 
Does this mean that the rounding mechanism (which rounded up the storage to the neared GiB) is not valid anymore? 

If users can provision any storage size, and the rounding mechanism does not update it to the nearest GiB, then we can remove this note.

Reply:
Hi yes this is correct. Concretely we have another change in flight that changes the minimum to the nearest rounded byte amount that is a multiple of 4096 bytes. 

Currently we use the byte amount as is (no rounding)

Reply:
@jakobmoellerdev Thanks for the confirmation.
Removed the note.